app/src/main/java/eu/kanade/presentation/manga/components/MangaInfoHeader.kt

```diff
@@ -164,6 +164,7 @@ fun MangaActionRow(
     onEditCategory: (() -> Unit)?,
     // SY -->
     onMergeClicked: () -> Unit,
+    showMergeButton: Boolean,
```

Comment:
Instead of a boolean make onMergeClicked nullable `onMergeClicked: (() -> Unit)?`

Reply:
https://github.com/jobobby04/TachiyomiSY/pull/653/commits/69e7a1a229dcbe58e439f5fc88262835a930b338